modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkActionDescriptor.java

```diff
@@ -38,7 +39,7 @@
 
     // @since 11.1
     @XNode("@enabled")
-    protected boolean isEnabled = true;
+    protected Boolean enabled;
```

Comment:
Aren't you changing the merge semantics? Previously if we had

    <extension target="" point="">
      <something enabled="false" />
      <something ... /> <!-- completely new contribution -->
    </extension>

Then the new contribution would be enabled, but now it'll keep a merged enabled=false flag. Or is something else changing in the merge?

Reply:
You're right, this one will keep the {{enabled=false}}. 

This change was introduced in 11.x, I think we can break the compatibility there, wdyt?

Reply:
I'm ok, I just wanted to be sure you were aware of and ok with the change.